zerver/tests/test_openapi.py

```diff
@@ -351,6 +351,18 @@ def get_standardized_argument_type(self, t: Any) -> Union[type, Tuple[type, obje
         else:  # nocoverage  # in python3.5. I.E. this is used in python3.6+
             origin = getattr(t, "__origin__", None)
 
+        if sys.version_info > (3, 6):  # nocoverage  # in < python3.7
```

Comment:
Can you rework this block to have a more clear if/else structure for versions 3.5, 3.6, >3.6?  It seems a bit unclean to have this right after an `else` block with the first line the same.

Reply:
Sure, happy to. I emulated what was already there, because I wasn't really sure if the comments were magical outside of the nocoverage.